api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -258,10 +258,11 @@ private RdrResearcher toRdrResearcher(DbUser dbUser) {
                       ? verifiedInstitutionalAffiliation.getInstitutionalRoleOtherText()
                       : roleEnum.toString();
 
-              new ResearcherVerifiedInstitutionalAffiliation()
-                  .institutionShortName(
-                      verifiedInstitutionalAffiliation.getInstitution().getShortName())
-                  .institutionalRole(role);
+              researcher.setVerifiedInstitutionalAffiliation(
```

Comment:
Not related to this Story but i noticed this information was not being send as part of exportResearcher hence updated it as well

Reply:
good catch, that seems important :)